server/doc.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+mongoose.promise = Promise;
+const Schema = mongoose.Schema;
+const date = require('date-and-time');
+
+const docSchema = new Schema({
+    name: {type: String, required = true},
+    file: {type: String, required = true},
+    owner_id: {type: int, required = true},
+    locked: int,
+    revision_id: int,
+    last_revised: {type: Date, required = true}
+})
+
+docSchema.methods.filter = function(file_text, taboo_words){
```

Comment:
This method does not change any of the values within a doc, i.e. this method does nothing.

Reply:
retrieved the attribute using this to modify it